Client/Frontend/Browser/TabManager.swift

```diff
@@ -378,66 +378,96 @@ class TabManager : NSObject {
     }
 }
 
-extension TabManager {
```

Comment:
Most of these deletions are from extracting SavedTab out of the TabManager scope.